gemviz23/demo/filterpanel.py

```diff
@@ -89,10 +89,15 @@ def filteredCatalog(self):
         if len(scan_id) > 0:
             cat = utils.get_tiled_runs(cat, scan_id=int(scan_id))
 
-        motors=self.positioner.text().strip()
+        motors=self.positioners.text().strip()
         if len(motors) > 0: 
-            for motor in motors.split(" "):
-                cat = cat.search(tiled.queries.Contains("motors", motor))
+            for motor in motors.split(","):
+                cat = cat.search(tiled.queries.Contains("motors", motor.strip()))
```

Comment:
why use `.strip()` inside `cat = cat.search(tiled.queries.Contains("motors", motor.strip()))` isn't the split method already taking care of removing the leading/trailing spaces?

Reply:
The first strip takes care of whitespace at the start and end of the string.  With the next `split(",")`, that could leave each individual motor with leading (or even, pathologically trailing) white space.  So, the additional `strip()`.

motors | code | result
--- | :--- | ---
"  m1  , m2  " | `motors.split(",")` | ["  m1  ", " m2"]
"  m1  , m2  " | `motors.strip().split(",")` | ["m1  ", " m2"]
"  m1  , m2  " | `[motor.strip() for motor in motors.strip().split(",")]` | ["m1", "m2"]